finders/metadata/cma-cases.json

```diff
@@ -3,12 +3,20 @@
   "base_path": "/cma-cases",
   "format_name": "Competition and Markets Authority case",
   "name": "Competition and Markets Authority cases",
+  "description": "Find reports and updates on current and historical CMA investigations",
   "filter": {
     "document_type": "cma_case"
   },
   "signup_content_id": "43dd2b13-93ec-4ca6-a7a4-e2eb5f5d485a",
   "signup_copy": "You'll get an email each time a case is updated or a new case is published.",
   "organisations": ["957eb4ec-089b-4f71-ba2a-dc69ac8919ea"],
+  "topics": [
```

Comment:
specialist_sectors is the legacy/deprecated name for them.  They're called topics in all the new parts of the stack, so I wanted to keep that naming here.  At the point we send them to publishing-api, they will also be called topics.